etc/devices/appro-gb2.dev

```diff
@@ -44,18 +44,6 @@ specification "sr5110" {
 		}
 		expect "-iSCB> "
 	}
-	script status {
-		send "pmnode %s\r"
-		expect "node([0-9]+): (on|off|n/a)"
```

Comment:
In the commit message for this, it would probably be good to mention that the current behavior is changing so that the untested singletons would start to get used, so this change just preserves the existing behavior.